src/api/variants/collection.js

```diff
@@ -139,6 +139,7 @@ module.exports = class VariantCollection extends EntityCollection {
 
         views.filter(f => !_.includes(usedViews, f.base)).forEach(viewFile => {
             const name = utils.slugify(viewFile.name.split(source.get('splitter'))[1]).toLowerCase();
+            const notes = resources.find(r => r.base === `${component.name}${source.get('splitter')}${name}.${source.get('files.notes')}.md`);
```

Comment:
this should be passed in via a separate parameter (like `views` is), since the variant readme files should not be considered resources at all.